index.js

```diff
@@ -140,7 +140,14 @@ let wrapCommand = function (fn, commandName, beforeCommand, afterCommand) {
             return future.wait()
         } catch (e) {
             futureFailed = true
+            console.log('futureFailed', e.stack)
+
+            // TODO: if fn() has already thrown an exception, why re-run it here?
```

Comment:
`fn` hasn't thrown an action. `future.wait()` did because there was Fiber context. This happens when you do this:
```js
browser.command(); // works
process.nextTick(() => {
    browser.command(); // doesn't work throws the exception there
})
```